src/jit/hwintrinsicxarch.cpp

```diff
@@ -219,19 +219,64 @@ unsigned Compiler::simdSizeOfHWIntrinsic(NamedIntrinsic intrinsic, CORINFO_SIG_I
 }
 
 //------------------------------------------------------------------------
-// numArgsOfHWIntrinsic: get the number of arguments
+// numArgsOfHWIntrinsic: get the number of arguments based on table and
```

Comment:
Maybe reword to:
 ```
gets the number of arguments for the hardware intrinsic.
This attempts to do a table based lookup but will fallback to the number of operands in 'node' if the table entry is -1.
```

Reply:
Sounds better that current one - will change. Next PR would be OK?

Reply:
If you are going to push any more changes, I would fix it this PR. Otherwise I think next PR is fine (provided @CarolEidt agrees).